src/createUrl.js

```diff
@@ -17,7 +17,11 @@
  * @return {string} - created URL
  */
 function createUrl(template, params) {
-  // write code here
+  const missingValuePattern = /{(\w+)}/g;
```

Comment:
у нас всего одна регулярка в функции, зачем усложнять?
```suggestion
  const pattern = /{(\w+)}/g;
```